client/src/components/Logout.tsx

```
@@ -1,7 +1,25 @@
+
+  let user = authContext?.user;
+
+  const handleLogout = () => {
+    user = null;
+    navigate('/login');
+  };
+
   return (
-    <div>Logout</div>
-  )
+    <button onClick={handleLogout}>
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)